packages/office-ui-fabric-react/src/components/DetailsList/examples/DetailsList.Grouped.Large.Example.tsx

```diff
@@ -25,29 +26,39 @@ const _columns: IColumn[] = [
   }
 ];
 
-const _items = Array.apply(null, Array(1000)).map((_: any, num: number) => ({
-  key: num.toString(),
-  name: `Item ${num.toString()}`,
-  value: num.toString()
-}));
+interface IDetailsListGroupedLargeExampleItem {
+  key: string;
+  name: string;
+  value: string;
+}
 
-const _groups = Array.apply(null, Array(10)).map((_: any, num: number) => ({
-  key: num.toString(),
-  name: num.toString(),
-  startIndex: num * 100,
-  count: 100
-}));
+let _items: IDetailsListGroupedLargeExampleItem[];
+let _groups: IGroup[];
 
-export class DetailsListGroupedLargeExample extends BaseComponent<{}, { items: {}[] }> {
+export class DetailsListGroupedLargeExample extends React.Component<{}, {}> {
   private _root = React.createRef<IDetailsList>();
 
   constructor(props: {}) {
     super(props);
+    if (!_items) {
+      _items = Array.apply(null, Array(1000)).map((_: any, num: number) => ({
+        key: num.toString(),
+        name: `Item ${num.toString()}`,
+        value: num.toString()
+      }));
+
+      _groups = Array.apply(null, Array(10)).map((_: any, num: number) => ({
+        key: num.toString(),
+        name: num.toString(),
+        startIndex: num * 100,
+        count: 100
+      }));
+    }
   }
 
   public render() {
     return (
-      <Fabric className="DetailsList-grouped-large-example">
+      <Fabric>
```

Comment:
You do *not* need to change this but I believe we should be able to remove `Fabric` wrapping component from examples.

Reply:
The Fabric wrapping component is still helpful if the example contains any text outside of a Fabric component (which granted this one doesn't...).
```
<Fabric>
  <div>text that will be in default font without Fabric wrapper</div>
  <DetailsList .../>
</Fabric>
```